src/main/java/gitflow/ui/GitflowWidget.java

```diff
@@ -46,16 +49,16 @@
 import git4idea.repo.GitRepository;
 import git4idea.repo.GitRepositoryChangeListener;
 import git4idea.ui.branch.GitBranchWidget;
-import gitflow.actions.GitflowActions;
 
 /**
  * Status bar widget which displays actions for git flow
  *
  * @author Kirill Likhodedov, Opher Vishnia, Alexander von Bremen-Kühne
  */
-public class GitflowWidget extends GitBranchWidget implements GitRepositoryChangeListener {
+public class GitflowWidget extends GitBranchWidget implements GitRepositoryChangeListener, StatusBarWidget.TextPresentation {
     private volatile String myText = "";
     private volatile String myTooltip = "";
+    private volatile boolean myGitFlowVersionSupported;
```

Comment:
No need to cache this here... it's already one version of Git Flow CLI per system and it's cached in GitflowVersionTester